packages/connect/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+### Changed
+
+- Stop using dynamic imports in order to import the "well-known" specs, due to the fact that not all bundlers are able to handle dynamic imports gracefully on package's code.
+
```

Comment:
Sure thing. I do not think we need to add this at all - as its internal things of SC that are not need to be known to the user;
```suggestion
```